lib/googleauth/service_account.rb

```diff
@@ -211,16 +210,23 @@ def updater_proc
       protected
 
       # Creates a jwt uri token.
-      def new_jwt_token jwt_aud_uri, options = {}
+      def new_jwt_token jwt_aud_uri = nil, options = {}
         now = Time.new
         skew = options[:skew] || 60
         assertion = {
           "iss" => @issuer,
           "sub" => @issuer,
-          "aud" => jwt_aud_uri,
           "exp" => (now + EXPIRY).to_i,
           "iat" => (now - skew).to_i
         }
+
+        if @scope && jwt_aud_uri
+          raise ArgumentError, "Cannot specify both scope and aud"
+        end
+
+        assertion["scope"] = @scope.join " " if @scope
```

Comment:
```suggestion
        assertion["scope"] = Array(@scope).join " " if @scope
```

So that it will still work if someone provides a single string rather than an array of strings as the `@scope`